core/sql/sqlcomp/CmpSeabaseDDLmd.h

```diff
@@ -238,6 +238,16 @@ static const QString seabaseObjectsUniqIdxDDL[] =
   {" ; "}
 };
 
+enum SeabaseObjectsFlags {
+  SEABASE_OBJECT_IS_EXTERNAL_HIVE  = 0x0000000000000001, // set if this object
```

Comment:
I generated this enum based off a similar FLAGS field for the COLUMNS table.  
num SeabaseColumnsFlags {
  SEABASE_COLUMN_IS_SALT       = 0x0000000000000001, // don't rely on this quite yet,                                                   
  SEABASE_COLUMN_IS_DIVISION   = 0x0000000000000002

I think we should be consistent in the naming.